RubeesEat/wwwroot/css/site.css

```diff
@@ -250,12 +250,18 @@ input[type="button"].button-primary:focus {
 }
 
 .entryLines {
+    display: flex;
+    flex-direction: column;
     list-style: none;
     padding: 0;
-    margin: 10px;
+    margin: 10px auto;
+    align-items: center;
 }
 
 .entryLines li {
+    max-width: 800px;
+    width: 100%;
+    word-wrap: break-word;
     margin: 10px 10%;
     padding: 10px;
     background-color: white;
```

Comment:
Why is the background color set to white here?

Reply:
This is for a general Entryline, that does not have a background color for positive/negative amount. This is for looking at an individual bill (where there is no positive/negative amount)